docs/code_ref/coordinates/rotatedsunframe.rst

```diff
@@ -0,0 +1,390 @@
+.. _sunpy-coordinates-rotatedsunframe:
+
+Differential rotation using coordinate frames
+=============================================
+
+Normally, coordinates refer to a point in inertial space (relative to the barycenter of the solar system).
+Transforming to a different observation time does not move the point at all, but rather only updates the coordinate representation as needed for the origin and axis orientations at the new observation time.
+However, the Sun both moves translationally in inertial space and rotates about its axis over time.
+Thus, a coordinate that points to a solar feature (e.g., the center of the Sun or an active region on its surface) will not continue to point to the same solar feature at other observation times.
+
+To evolve a coordinate in time such that it accounts for the rotational motion of the Sun, one can use the `~sunpy.coordinates.metaframes.RotatedSunFrame` "metaframe" class as described below.
+This machinery will take into account the latitude-dependent rotation rate of the solar surface, also known as differential rotation.
+
+In addition, one may want to account for the translational motion of the Sun as well, and that can be achieved by also using the context manager `~sunpy.coordinates.transform_with_sun_center` for desired coordinate transformations.
+
+Basics of the RotatedSunFrame class
+-----------------------------------
+In a nutshell, the `~sunpy.coordinates.metaframes.RotatedSunFrame` class allows one to specify coordinates in a coordinate frame prior to an amount of solar (differential) rotation being applied.
+That is, the coordinate will point to a location in inertial space at some time, but will use a coordinate system at a *different* time to refer to that point, while accounting for the differential rotation between those two times.
+
+Technical note: ``RotatedSunFrame`` is not itself a coordinate frame, but is instead a "metaframe".
+A new frame class is created on the fly corresponding to each base coordinate frame class.
+This tutorial will refer to these new classes as ``RotatedSun*`` frames.
+
+Creating coordinates
+^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
There are lots of class links here that should be intersphinx links and not double-dashes. Happy for another issue to be opened to fix this though.